oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -2345,11 +2339,6 @@ private boolean withClientSession() {
         return connection.getStatus().isClientSessionSupported() && useClientSession;
     }
 
-    private boolean secondariesWithinAcceptableLag() {
-        return getClient().getReplicaSetStatus() == null
-                || connection.getStatus().getReplicaSetLagEstimate() < acceptableLagMillis;
```

Comment:
`acceptableLagMillis` should be marked as deprecated since it won't be used anymore.